ContactUI.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.0rc1/angular-route.min.js"></script>
+
+</head>
+<body ng-app="myApp">
+
+<div class="w3-container">
+    <div class="w3-bar w3-teal" ng-controller="contactsCtrl">
+        <div class="w3-bar-item w3-teal">
+            <button  class="w3-btn w3-teal w3-ripple w3-border-green" ng-click="goToHome('/home')">Contacts</button>
+
+        </div>
+        <div class="w3-bar-item w3-teal">
+            <button  class="w3-btn w3-teal w3-ripple w3-border-green" ng-click="goToEdit('new')">Create New Contact</button>

```

Comment:
instead of having functions for navigation, you can use something like:
```html
data-ng-href="#/home/"
data-ng-hreg="#/contacts/new"
data-ng-hreg="#/contacts/12"
```